CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [0.0.4] - 2019-11-02
+
+### Added
+* Added recent update to Speck library to do with jumpy behaviour on click-and-drag.
```

Comment:
```suggestion
* Added recent update to Speck library to fix jumpy behavior on click-and-drag.
```
I personally use Canadian spelling, but all the Plotly docs and materials use American spelling, so...